src/main/java/tutoraid/logic/parser/TutorAidParser.java

```diff
@@ -42,30 +38,20 @@ public Command parseCommand(String userInput) throws ParseException {
         final String commandWord;
         final String arguments;
 
-        if (userInput.contains("-s")) {
-            String[] extracts = userInput.split("-s", 2);
-            commandWord = extracts[0] + "-s";
-            arguments = extracts[1];
-        } else if (userInput.contains("-p")) {
-            String[] extracts = userInput.split("-p", 2);
-            commandWord = extracts[0] + "-p";
-            arguments = extracts[1];
-        } else {
-            matcher = BASIC_COMMAND_FORMAT.matcher(userInput.trim());
-            if (!matcher.matches()) {
-                throw new ParseException(String.format(
-                        Messages.MESSAGE_INVALID_COMMAND_FORMAT, HelpCommand.MESSAGE_USAGE));
-            }
-            commandWord = matcher.group("commandWord");
-            arguments = matcher.group("arguments");
+        matcher = BASIC_COMMAND_FORMAT.matcher(userInput.trim());
+        if (!matcher.matches()) {
+            throw new ParseException(String.format(
+                    Messages.MESSAGE_INVALID_COMMAND_FORMAT, HelpCommand.MESSAGE_USAGE));
         }
+        commandWord = matcher.group("commandWord");
+        arguments = matcher.group("arguments");
 
         switch (commandWord) {
 
-        case AddStudentCommand.COMMAND_WORD:
-            return new AddStudentCommandParser().parse(arguments);
+        case AddCommandParser.COMMAND_WORD:
```

Comment:
I was thinking this might not be consistent with other cases where the `COMMAND_WORD` comes from `Command` classes instead of `Parser` classes.